configs/mmpose/pose-detection_rtmo_onnxruntime_dynamic.py

```diff
@@ -0,0 +1,25 @@
+_base_ = ['./pose-detection_static.py', '../_base_/backends/onnxruntime.py']
```

Comment:
This config seems same as `configs/mmpose/pose-detection_yolox-pose_onnxruntime_dynamic.py`, shall we use `configs/mmpose/pose-detection_yolox-pose_onnxruntime_dynamic.py`?

Reply:
The `post_processing` parameters are slightly different. Additionally, users may be confused about which configuration to use for deployment if they do not read the document.